frontend/src/App.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import NavBar from './NavBar';
```

Comment:
nit: can we make `import`s consistent with either single quote or double quotes?